eng/Packages.Data.props

```diff
@@ -87,6 +87,10 @@
     <PackageReference Update="NSubstitute" Version="3.1.0" />
     <PackageReference Update="nunit" Version="3.12.0" />
     <PackageReference Update="NUnit3TestAdapter" Version="3.13.0" />
+    <PackageReference Update="OpenTelemetry" Version="0.6.0-beta.1" />
+    <PackageReference Update="OpenTelemetry.Extensions.Hosting" Version="0.6.0-beta.1" />
+    <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="0.6.0-beta.1" />
+    <PackageReference Update="OpenTelemetry.Instrumentation.Http" Version="0.6.0-beta.1" />
```

Comment:
@pakrym This is the change we discussed. Please confirm that this is appropriate.

Reply:
LGTM!